Source/Border.js

```diff
@@ -112,6 +112,74 @@ export class Border extends WrappedObject {
     this.sketchObject.gradientGeneric().gradientType = gradientTypeEnum;
     this.sketchApplication.reloadInspector();
   }
+  /**
+  Return a list of tests to run for this class.
+
+  @return {dictionary} A dictionary containing the tests to run. Each key is the name of a test, each value is a function which takes a Tester instance.
+  */
+
+  /**
+   Return a list of tests to run for this class.
+
+   @return {dictionary} A dictionary containing the tests to run. Each key is the name of a test, each value is a function which takes a Tester instance.
+   */
+
+  static tests() {
+    return {
+      "tests" : {
+          "testBorderThickness" : function(tester) {
+            var border = new Border()
+            border.thickness = 4;
+            tester.assertEqual(border.thickness, 4)
+          },
+
+          "testBorderPosition" : function(tester) {
+            var border = new Border()
+
+            border.positions = borderPosEnum["outside"];
+            tester.assertEqual(border.positions, 2)
+
+            border.positions = borderPosEnum["inside"];
+            tester.assertEqual(border.positions, 1)
+
+            border.positions = borderPosEnum["center"];
+            tester.assertEqual(border.positions, 0)
+          },
+
+          "testGradientTypes" : function(tester) {
+            var border = new Border()
+
+            border.gradientType = ColorHelper.gradientTypeEnum["flat"];
```

Comment:
Another opinion ;) my preference would be to remove the `Enum` from them, but to be verbose for clarity: `position` instead of `pos`. (This is my Obj-c/Swift developer mindset talking :)

```javascript
border.positions = borderPosition.outside
border.positions = borderPosition.center

ColorHelper.gradientType.flat
ColorHelper.gradientType.linear
ColorHelper.gradientType.radial
```

Also, why are the two enums used differently? It looks like `borderPosEnum` is a global object? If that's the base it probably should be capitalized: `BorderPosition`. But then why is gradient type different, being hung off of `ColorHelper`?